6_3_iterable.js

```diff
@@ -0,0 +1,25 @@
+class Group {
+    constructor(arr) {
+        this.cluster = arr;
+    }
+    add = value => (this.cluster.includes(value) ? null : this.cluster.push(value));
+    delete = value =>
+        this.cluster.includes(value) ? this.cluster.splice(this.cluster.indexOf(value), 1) : null;
+    has = value => this.cluster.some(item => item === value);
+
+    static from(arr) {
+        return new Group(arr);
+    }
+
+    [Symbol.iterator]() {
+        let data = this.cluster
```

Comment:
You still haven't installed the Prettier extension? 😅

Reply:
I've done that 😅 Even I have .prettierrc file